contracts/SpokePool.sol

```diff
@@ -160,15 +168,15 @@ abstract contract SpokePool is
         uint256[] refundAmounts,
         uint32 indexed rootBundleId,
         uint32 indexed leafId,
-        address l2TokenAddress,
-        address[] refundAddresses,
+        bytes32 l2TokenAddress,
+        bytes32[] refundAddresses,
         address caller
```

Comment:
Should we make `caller` a `bytes32` here for consistency with other events?

Reply:
I think yes, that would be more consistent